src/index.html

```diff
@@ -5,8 +5,465 @@
+           Product code: 195434
+        </p>
+
+        <div class="card__item-rating">
+          <ul class="stars stars--4">
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+          </ul>
+          <p class="card__reviews">
+           Reviews: 5
+          </p>
+        </div>
+
+        <div class="card__item-price">
+
+          <p class="card__price-text">
+           Price:
+          </p>
+          <p class="card__price-number">
+           $2,199
+          </p>
+
+        </div>
+
+        <a href="#"
+         class="card__buy-button"
+         data-qa="card-hover"
+       >
+         BUY
+        </a>
+
+      </div>
+
+      <div
+      class="card main__card"
+      data-qa="card"
+     >
+        <img class="card__imac-image"
+         src="./images/imac.jpeg"
+         alt="imac"
+       >
+
+
+        <p class="card__item-name">
+           APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+
+        <p class="card__item-code">
+           Product code: 195434
+        </p>
+
+        <div class="card__item-rating">
+          <ul class="stars stars--4">
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+          </ul>
+          <p class="card__reviews">
+           Reviews: 5
+          </p>
+        </div>
+
+        <div class="card__item-price">
+
+          <p class="card__price-text">
+           Price:
+          </p>
+          <p class="card__price-number">
+           $2,199
+          </p>
+
+        </div>
+
+        <a href="#"
+         class="card__buy-button"
+         data-qa="card-hover"
+       >
+         BUY
+        </a>
+
+      </div>
+
+      <div
+      class="card main__card"
+      data-qa="card"
+     >
+        <img class="card__imac-image"
+         src="./images/imac.jpeg"
+         alt="imac"
+       >
+
+
+        <p class="card__item-name">
+           APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+
+        <p class="card__item-code">
+           Product code: 195434
+        </p>
+
+        <div class="card__item-rating">
+          <ul class="stars stars--4">
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+          </ul>
+          <p class="card__reviews">
+           Reviews: 5
+          </p>
+        </div>
+
+        <div class="card__item-price">
+
+          <p class="card__price-text">
+           Price:
+          </p>
+          <p class="card__price-number">
+           $2,199
+          </p>
+
+        </div>
```

Comment:
Please, apply same styling pattern through out your html (one empty line between siblings, no empty line between parent and a child)
```suggestion
        <div class="card__item-price">
          <p class="card__price-text">
           Price:
          </p>
          
          <p class="card__price-number">
           $2,199
          </p>
        </div>
```